frontend/app/api/players/route.js

```diff
@@ -1,10 +1,21 @@
-import { NextResponse } from "next/server";
-import postgres from "postgres";
+import { NextResponse } from 'next/server';
+import postgres from 'postgres';
 
-const sql = postgres("postgresql://piko:pikopw@localhost:5432/piko-db?ssl=false");
+const sql = postgres('postgresql://piko:pikopw@localhost:5432/piko-db?ssl=false', {
+  idle_timeout: 20,
+  max_lifetime: 60 * 30
+});
 
 export async function GET() {
   return NextResponse.json(await sql`
     SELECT * FROM players;
   `);
 }
+
+export async function POST(request) {
+  const { pname } = await request.json();
+  return NextResponse.json(await sql`
+    INSERT INTO players (pname)
+    VALUES (${pname});
+  `);
+}
```

Comment:
can you add the ID fix to the seeder. for the other tables too?
`SELECT setval(pg_get_serial_sequence('players', 'id'), coalesce(max(id)+1, 1), false) FROM players;`
`SELECT setval(pg_get_serial_sequence('matches', 'id'), coalesce(max(id)+1, 1), false) FROM matches;`